srcpkgs/river/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'river'
 pkgname=river
-version=0.2.3
+version=0.2.4
 revision=1
 build_style=zig-build
-configure_args="$(vopt_if xwayland -Dxwayland) -Dpie"
+configure_args="$(vopt_if xwayland -Dxwayland) -Dpie -Dstrip"
 hostmakedepends="pkg-config wayland-devel scdoc"
-makedepends="wlroots-devel libevdev-devel"
+makedepends="wlroots-devel libevdev-devel pixman-devel
+ wayland-protocols libxkbcommon-devel wayland-devel"
 depends="$(vopt_if xwayland xorg-server-xwayland)"
 short_desc="Dynamic tiling Wayland compositor"
 maintainer="Isaac Freund <mail@isaacfreund.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/riverwm/river"
-distfiles="${homepage}/releases/download/v${version}/river-${version}.tar.gz"
-checksum=7acdd2b929e16a20006b4baf1f29e7a8dc7f70cf5747e1503a610a90d1637132
+distfiles="https://github.com/riverwm/river/releases/download/v${version}/river-${version}.tar.gz"
+checksum=26c1c41a65ce3804069afad6988410515cf478d2b76303ebc699766d3d4dc69f
 
 build_options="xwayland"
-build_options_default="xwayland"
```

Comment:
Why change this default? Most users will want Xwayland likely and those that know what it is and know they don't want it shouldn't have much trouble disabling it.

Reply:
i thought pure wayland could be the default by now..

Reply:
I am not sure I understand why people wouldn't want xwayland support in their compositor.

Reply:
since wayland support is more mature, one could argue that having xwayland enabled is less secure and not worth it.

Reply:
for example, i don't need to worry about https://github.com/void-linux/void-packages/issues/43174